tests/integration/mixer/policy/ratelimiting_test.go

```diff
@@ -112,6 +120,68 @@ func TestRateLimiting_DefaultLessThanOverride(t *testing.T) {
 		})
 }
 
+func TestRateLimiting_ExternalTraffic(t *testing.T) {
+	framework.Run(t, func(ctx framework.TestContext) {
+		destinationService := "istio-egressgateway"
+
+		bookinfoNs, g, red, _, prom := setupComponentsOrFail(t, ctx)
```

Comment:
curious: why bookinfo?  does this need to have anything to do with bookinfo?

Reply:
Need curl installed in app and we already deploy bookinfo as part of other ratelimiting tests...

Reply:
Does the setup carry over between tests?  I don't know the new framework well enough to judge.

There is a basic sleep container with curl iirc and we have the fortio container that we could use as an alternative.

Reply:
Yes, there is a testsetup thats deploys bookinfo and redis and that is carried over through all the tests